index.html

```diff
@@ -4,36 +4,50 @@
-      First protected in 1864, Yosemite National Park is best known for its waterfalls, but within its nearly 1,200 square miles, you can find deep valleys, grand meadows, ancient giant sequoias, a vast wilderness area, and much more.
-      
-      "https://image.flaticon.com/icons/svg/2979/2979348.svg"
-      Fire is a natural and essential part of Yosemite. We manage fire carefully and study how it interacts with the park’s ecosystems. This blog provides updates about fires in Yosemite. Check for current fire restrictions.
-      
-      "https://image.flaticon.com/icons/svg/820/820374.svg"
-      We strongly recommend that you make reservations for lodging, camping, and backpacking. You can pay the park entrance fee upon arrival (there's no need to pay it in advance).
+  <nav>
+    <div class="navContainer">
+    <div class="rednav"><a href="#">Plan your Visit</a></div>
+    <div class="bluenav"><a href="#">Learn About the Park</a></a></div>
+    <div class="yellownav"><a href="#">Get Involved</a></div>
+    </div>
+  </nav>     
+  <header>
+    <div class="header_img">
+      <img src="assets/header_img.jpg" alt="Yosemite Landscape">
+    </div>
+    <div class="header_txt">
+      <h1>Yosemite Park Guide</h1>
+      <h2>Come explore California’s beautiful wilderness.</h2>
+      <h3>Always open. Reservations are not available or required to enter the park.</h3>
+    </div>
+  </header>
+  <section class="sectionOne">
+    <div class="textBox">
+    <p>Not just a great valley, but a shrine to human foresight, the strength of granite, the power of glaciers, the persistence of life, and the tranquility of the High Sierra.</p>
+      
+    <p>First protected in 1864, Yosemite National Park is best known for its waterfalls, but within its nearly 1,200 square miles, you can find deep valleys, grand meadows, ancient giant sequoias, a vast wilderness area, and much more.</p>
+  </div>
+  </section>
+  <section class="sectionTwo">
+    <img src="https://image.flaticon.com/icons/svg/2979/2979348.svg" alt="fire_icon">
+    <p>Fire is a natural and essential part of Yosemite. We manage fire carefully and study how it interacts with the park’s ecosystems. This blog provides updates about fires in Yosemite. Check for <span>current fire restrictions.</span></p>
     
-      "footer_img.png"
-      
-      
-        Plan your Visit
-        Learn about the Park
-        Get Involved
-      
+    <img src="https://image.flaticon.com/icons/svg/820/820374.svg" alt="lodging_icon">
+    <p>We strongly recommend that you make reservations for lodging, camping, and backpacking. You can pay the <span>park entrance fee </span>upon arrival (there's no need to pay it in advance).</p>
+  </section>
+  <footer>
+    <div class="footer-image">
+      <img src="assets/footer_img.jpg">
+    </div>
+  <nav>
+    <a class="rednav" href="#">Plan your Visit</a>
+    <a class="bluenav" href="#">Learn about the Park</a>
+    <a class="yellownav" href="#">Get Involved</a>
```

Comment:
<div class="yellownav"><a href="#">Get Involved</a></div> wrap it in a div like the nav from the header and move the class to the div to fix that pesky little footer nav link. Not sure why the others still looked fine 🤷‍♂️